### PR TITLE
Prevent self-handling of flags, log slowmode=0

### LINKED ISSUE #93: Prevent moderators from clearing flags about themselves

### PR DESCRIPTION
**Pull request by BobbyTheCatfish**
Resolves #93 and #141 

Since its a small enough change, I've also included a fix for #95

### CODE REVIEW

File: modules/01-filter.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ async function processCardAction(interaction) {
     const mod = interaction.member,
       embed = u.embed(flag.embeds[0]),
       infraction = await Module.db.infraction.getByFlag(flag);
-
+    if (mod.id == infraction.discordId) return interaction.reply({ content: "You can't handle your own flag!", ephemeral: true });
```
Comment (LaughLax) [inline modules/01-filter.js:172]: Returning here without removing the card from `processing` will lock all mods from taking action on the card until an Icarus reboot.

File: modules/01-mods.js [hunk 2]
```diff
@@ -280,13 +280,20 @@ async function slashModSlowmode(interaction) {
 
   if (duration <= 0) {
     ch.edit({ rateLimitPerUser: 0 }).catch(e => u.errorHandler(e, interaction));
-
+    const old = molasses.get(ch.id);
     if (molasses.has(ch.id)) {
       clearTimeout(molasses.get(ch.id));
       molasses.delete(ch.id);
```
Comment (LaughLax) [inline modules/01-mods.js:286]: May as well use the `old` variable you created in these lines, to DRY things up

**Review by LaughLax — CHANGES_REQUESTED** [review]
See line comments

File: modules/01-filter.js (hunk 1 above)
Comment (BobbyTheCatfish) [reply modules/01-filter.js:172]: Fixed in latest commit

File: modules/01-mods.js (hunk 2 above)
Comment (BobbyTheCatfish) [reply modules/01-mods.js]: Fixed in latest commit

File: modules/01-mods.js [hunk 3]
```diff
@@ -280,13 +280,20 @@ async function slashModSlowmode(interaction) {
...
   if (duration <= 0) {
     ch.edit({ rateLimitPerUser: 0 }).catch(e => u.errorHandler(e, interaction));
-
-    if (molasses.has(ch.id)) {
-      clearTimeout(molasses.get(ch.id));
-      molasses.delete(ch.id);
+    const old = molasses.get(ch.id);
+    if (old) {
+      clearTimeout(old);
+      molasses.delete(old);
```
Comment (LaughLax) [inline modules/01-mods.js:286]: This should still be `ch.id`

File: modules/01-filter.js [hunk 4]
```diff
@@ -164,12 +164,15 @@ async function processCardAction(interaction) {
       await interaction.reply({ content: "Someone is already processing this flag!", ephemeral: true });
       return;
     }
-    processing.add(flag.id);
```
Comment (LaughLax) [inline modules/01-filter.js:167]: This needs to stay the very first thing that happens (especially before any `await` calls) to prevent race conditions.

**Review by LaughLax — CHANGES_REQUESTED** [review]

**Comment by BobbyTheCatfish** [comment]
changes have been made

**Review by LaughLax — APPROVED** [review]

**Pull request merged**